packages/babel-plugin-proposal-class-properties/src/index.js

```diff
@@ -245,6 +299,77 @@ export default declare((api, options) => {
       });
   }
 
+  function buildClassStaticPrivatePropertySpec(
+    ref,
+    path,
+    state,
+    privateClassId,
+  ) {
+    const { scope, parentPath } = path;
+    const { key, value } = path.node;
+    const { name } = key.id;
+    const privateId = scope.generateUidIdentifier(name);
```

Comment:
Nit: So we actually don't need to generate a unique id here, we can just use `name`.